src/assessment.js

```diff
@@ -27,7 +27,7 @@ var q1 = /* your answer here */;
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-var q2 = [/* your answer here */];
+var q2 = [fire];
```

Comment:
I guess I didn't understand the question correctly. My thinking was that since deathStar is defined in the global scale, so the variable itself is global and cannot access laserQuantity. But if it is IN the deathStar scope like laserSize, then I agree...